app/ui/client/views/app/room.js

```diff
@@ -215,8 +216,8 @@ Template.roomOld.helpers({
 				ts: 1,
 			},
 		};
-
-		return ChatMessage.find(query, options);
+		const result = ChatMessage.find(query, options).fetch().filter((msg) => room.t !== 'e' || msg.expireAt > now || msg.t);
```

Comment:
1. This can be on multiple lines
2. Why don't you pass the 't' as a filter to `query` instead of doing it in memory? 👀 
Something like
```javascript
query = { ...query, t: { $ne: 'e' } }
```

Reply:
I think making a different query for ephemeral rooms would be better. What do you think?